tests/test_rotation.py

```diff
@@ -0,0 +1,47 @@
+from unittest import TestCase
+import numpy as np
+
+from scipy.spatial.transform import Rotation as sp_rot
+from aspire.utils.rotation import Rotation
+from aspire.utils.types import utest_tolerance
+
+
+class UtilsTestCase(TestCase):
+    def setUp(self):
+        self.dtype = np.float32
+        self.rot_obj = Rotation(32, seed=0, seq='ZYZ', dtype=self.dtype)
```

Comment:
Maybe get rid of the 'ZYZ' when calling the constructor in both places (since one of the motivations of this class was to hide that stuff under the rug).

Reply:
Removed in commit 707aace.

Reply:
`testMultiplication` still has this

Reply:
Done in commit 948928f.